Xamarin.Forms.Core/Button.cs

```diff
@@ -49,9 +49,19 @@ public class Button : View, IFontElement, ITextElement, IBorderElement, IButtonC
 
 		public static readonly BindableProperty ImageProperty = ImageElement.FileImageProperty;
 
-
 		public static readonly BindableProperty PaddingProperty = PaddingElement.PaddingProperty;
 
+		public static readonly BindableProperty LineBreakModeProperty = BindableProperty.Create(nameof(LineBreakMode), typeof(LineBreakMode), typeof(Button), LineBreakMode.WordWrap,
+			propertyChanged: (bindable, oldvalue, newvalue) => ((Button)bindable).InvalidateMeasureInternal(InvalidationTrigger.MeasureChanged));
+
+		public static readonly BindableProperty MaxLinesProperty = BindableProperty.Create(nameof(MaxLines), typeof(int), typeof(Button), -1, propertyChanged: (bindable, oldvalue, newvalue) =>
+		{
+			if (bindable != null)
```

Comment:
if this is null, I think we can crash, there's nothing in the world to recover from.

Reply:
I think it will just do nothing when it's null. I can be wrong though. :-)

Reply:
Right. Maybe something like:

`if ((bindable is Button button)`

Fit better here?